### PR TITLE
Uma

### PR DESCRIPTION
**Pull request by usurakod**
Adding validation for DateTime and seats

### CODE REVIEW

File: src/main/java/com/app/katchup/Meeting/MeetingService.java [hunk 1]
```diff
@@ -18,17 +19,26 @@
...
+                    throw new Exception("startDateTime is before EndDateTime or date is not valid");
+            }
         }
         else{
             throw new EntityExistsException("Sorry! Meeting cannot be created. There exists another meeting on the given data");
         }
     }
 
+    public MeetingService() {
+        super();
```
Comment (sivajrm) [inline src/main/java/com/app/katchup/Meeting/MeetingService.java]: what is this for ? 😕

**Review by sivajrm — COMMENTED** [review]
Please look into the MeetingService and let me know why there is MeetingService() in line#33.
If that is not needed, please remove and push it. 
I will approve it.

File: src/main/java/com/app/katchup/Meeting/MeetingService.java (hunk 1 above)
Comment (usurakod) [reply src/main/java/com/app/katchup/Meeting/MeetingService.java]: Actually I didnt put that, Anyway will remove

**Review by sivajrm — APPROVED** [review]

**Comment by sivajrm** [comment]
With validation for PUT /response on host giving POLL, GO_WITH_MAJORITY options and code refactoring.
Added support for external participant user validation

**Pull request merged**